src/globals.py

```diff
@@ -8,12 +8,7 @@
 
 # The possible status an User can have
 possible_status = {'away', "vacation", "active"}
-status = {}
 
-# FIXME --> do that dynamically
-# the commands
-extensions = ['ping', 'clear', 'love', 'setstatus', 'survey', 'kick', 'roll']
 
 MESSAGE_SLEEP_TIME = 2
 MOD_LOG_CHANNEL_NAME = 'mod_log'
-MAX_CLEAR_NUM = 100
```

Comment:
Removed as discord's max clear number is 100, no need to store as global var as this is unlikely to change